dev/devicelab/bin/tasks/plugin_lint_mac.dart

```diff
@@ -48,8 +48,6 @@ Future<void> main() async {
         await _tryMacOSLint(macosintegrationTestPodspec, <String>[]);
       });
 
-      section('Create Objective-C plugin');
```

Comment:
I think this whole section needs to be delete, not just the section title. Anything referencing obj in this file, or migrating it to Swift if it still needs to be tested.

Reply:
done